src/main/scala/beam/agentsim/scheduler/BeamAgentScheduler.scala

```diff
@@ -131,7 +131,7 @@ class BeamAgentScheduler(
   private var startSender: ActorRef = _
   private var nowInSeconds: Int = 0
 
-  private val triggerMeasurer: TriggerMeasurer = new TriggerMeasurer
+  private val triggerMeasurer: TriggerMeasurer = new TriggerMeasurer(beamConfig.beam.debug.triggerMeasurer)
```

Comment:
While I understand the intent of pushing this into the object itself, and maybe keep it there as a backup - but I'd say why even create the object if it's disabled?

```
    private val triggerMeasurer: Option[TriggerMeasurer] = 
      if(beamConfig.beam.debug.triggerMeasurer.enabled) Some(new TriggerMeasurer(beamConfig.beam.debug.triggerMeasurer)) 
      else None
```

Reply:
In fact if my comment below is taken then the instantiation isn't even needed at all maybe?

Reply:
Fixed